<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>560</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>820</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>지식인 검색기</string>
  </property>
  <widget class="QWebEngineView" name="view_web" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>380</width>
     <height>511</height>
    </rect>
   </property>
   <property name="url" stdset="0">
    <url>
     <string>https://nid.naver.com/nidlogin.login?url=https%3A%2F%2Fkin.naver.com%2Findex.nhn</string>
    </url>
   </property>
  </widget>
  <widget class="QTableWidget" name="tb_posts">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>200</width>
     <height>490</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>제목</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>지식인 게시글</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tb_keywords">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>30</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>키워드</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btn_keyword">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>240</y>
     <width>90</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>편집</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>530</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>비우기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>검색 키워드</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tb_answers">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>310</y>
     <width>200</width>
     <height>210</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>90</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>90</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>키워드</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>내용</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btn_answer">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>530</y>
     <width>90</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>편집</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>290</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>답변 복사</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>검색 시작</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="pb_web">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>530</y>
     <width>391</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_keyword_close">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>240</y>
     <width>90</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>삭제</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_answer_close">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>530</y>
     <width>90</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>삭제</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
